Lib/test/test_subprocess.py

```diff
@@ -347,6 +347,18 @@ def test_cwd(self):
         temp_dir = self._normalize_cwd(temp_dir)
         self._assert_cwd(temp_dir, sys.executable, cwd=temp_dir)
 
+    def test_cwd_with_pathlike(self):
+        # Check that cwd changes the cwd for the child process with
+        # `os.PathLike`.
+        temp_dir = tempfile.gettempdir()
+        temp_dir = self._normalize_cwd(temp_dir)
+
+        class _PathLikeObj:
```

Comment:
@sayanchowdhury and I were discussing that at the PyCon Pune sprints and I suggested doing it this way to avoid adding a new dependency to the test cases.